d/desktop-file-utils/stone.yml

```diff
@@ -0,0 +1,24 @@
+#
+# SPDX-FileCopyrightText: © 2020-2024 Serpent OS Developers
+#
+# SPDX-License-Identifier: MPL-2.0
+#
+name        : desktop-file-utils
+version     : 0.27
+release     : 1
+homepage    : https://www.freedesktop.org/software/desktop-file-utils/releases
```

Comment:
I think this needs to be https://www.freedesktop.org/wiki/Software/desktop-file-utils/ - sorry!